tango_simlib/examples/override_class.py

```diff
@@ -3,16 +3,20 @@
 #                                                                                       #
 # BSD license - see LICENSE.txt for details                                             #
 #########################################################################################
-"""
-An example of the user-defined override class.
-"""
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
+"""An example of the user-defined override class."""
+from __future__ import absolute_import, division, print_function
 
 import logging
+from builtins import object
+
+from future import standard_library
+from past.builtins import cmp
+from PyTango import DevState, ErrSeverity, Except
+
+standard_library.install_aliases()
```

Comment:
Please fix the imports as per [wiki](https://github.com/ska-sa/katdeveloper/wiki/Investigate-a-way-to-improve-Py3-migration-feedback-loop-on-tox#futurize-stage-2)